docs/PayPalNativePayments/README.md

```diff
@@ -75,12 +75,43 @@ payPalNativeClient.listener = object : PayPalNativeCheckoutListener {
     override fun onPayPalCanceled() {
        // the user canceled the flow
     }
+}
+```
+
+You can optionally set the `shippingListener` on the `PayPalNativeCheckoutClient` to receive notifications when the user updates their shipping address or shipping method details.
+
+```kotlin
+payPalNativeClient.shippingListener = object : PayPalNativeShippingListener {
 
-    override fun onPayPalCheckoutShippingChange(
-        shippingChangeData: ShippingChangeData,
-        shippingChangeActions: ShippingChangeActions
+    override fun onPayPalNativeShippingAddressChange(
+        actions: PayPalNativePaysheetActions,
+        shippingAddress: PayPalNativeShippingAddress
     ) {
-        // there has been a change in shipping address or shipping method. Patch the order accordingly
+        // called when the user updates their chosen shipping address
+
+        // OPTIONAL: if you patch your order server-side, call actions.approve() or actions.reject() once complete.
```

Comment:
i would word this diferently, it could be misinterpreted that calling `approve` or `reject` is optional, when they are mandatory. Patching the order is optional.